Flood data visualization - Complete

dlm589 opened the pull request:
Hi Jeff, 

The functions are fully complete, including:
1. choropleth map showing the number of layers each municipality has
2. drop down menu to filter data, the filter data will show the selected jurisdiction first, then all the other associated jurisdictions below. 
3. clicking on the map layer will show municipal data first, then related regional and conservation authority data. 
4. Search by address is working, and will filter by the entered municipality. 

Next Step:
1. Nidhi and her team to provide more data for the missing municipalities. 
2. Nidhi and her team to re-organize some data, right now they have multiple entry of the same data because the data is from conservation authorities and spans over multiple municipalities. I recommended to put these data under the conservation authority bucket, it will be less manually intensive and less prone to error.

dlm589 committed:
Preliminary Setup, will change later

dlm589 committed:
Updated File Structure

dlm589 committed:
Updated file structure

dlm589 committed:
File Structure update

dlm589 committed:
Revert "Updated file structure"

This reverts commit e40c7742e6523daf7494021cdc363c21ba59feb9.

dlm589 committed:
Update file structure

dlm589 committed:
Added Map and Layers - Set up Link to Google Sheet

dlm589 committed:
Cleaned out files

dlm589 committed:
Added more layers

dlm589 committed:
Updated some codes

dlm589 committed:
Update to add address search

dlm589 committed:
Update Width

dlm589 committed:
Update code

dlm589 committed:
Updated Code to Enhance Performance

dlm589 committed:
Update +page.svelte

dlm589 committed:
Update +page.svelte

dlm589 committed:
Updated Code

Need to figure out the filters

dlm589 committed:
Code updates

dlm589 committed:
Code and Geojson Update

dlm589 committed:
Ready for Check In

dlm589 committed:
A small update that clears other drop down values when one jurisdiction is selected.

dlm589 committed:
Please Review

@jamaps please review this pull request, all issues have been resolved.

dlm589 committed:
Some bugs Fixed

@jamaps just fixed some bugs in the code. Now ready for review.

jamaps reviewed (changes requested):
Looks great! Many thanks!

A few little things to check/do then should be good to go
- Add Greater Toronto Area to title or as a subtitle (yes we know, but its good for non-locals to be able to understand)
- if click a municipality, remove the red dot for the address / or when address is selected, remove red outline of municipality
- the search bar, change to font to roboto/sans-serif (the trade gothic is hard to read when small), also change i.e. to e.g.
- probably a data error, but the 1 layer for York region is actually a Toronto Conservation Area layer - same with a few when I select City of Toronto - double check this to see why
- the GeoJSON data is too large, try to reduce the size so it can load easier for folks with lower bandwidth (reduce precision on coordates to 5 digits, delete not needed columns, and can reduce column names to one character - if still 5+mb we can think about simplifying the geometry)

dlm589 committed:
Ready for Deployment

I think it is ready for deployment, but it kept showing "Could not detect a supported production environment. See https://kit.svelte.dev/docs/adapters to learn how to configure your app to run on the platform of your choosing"
